### ISSUE TITLE
USER ISSUE: Stockpiles missing

### ISSUE THREAD

**Issue by Mephiztopheles** [StrangeLoopGames/EcoIssues#11434]
**Version:** 0.8.0.3 beta

**Steps to Reproduce:**
i dont know... i went away into the jungle on a self hosted server... came back. opened the inventory from the little wooden cart and moved the clay to the stockpiles... now i saw, that the items are placed on the ground and the stockpiles are missing. 

**Expected behavior:**
- the stockpiles should not destroy themself
- Items placed on the stockpiles should be displayed  correctly

**Actual behavior:**
- stockpiles destroy themself... maybe collected by GC
- items placed on the stockpiles are not or partly visible

**Do you have mods installed? Does issue happen when no mods are installed?:**
- stockpiles destroy themself... maybe collected by GC
- items placed on the stockpiles are not visible or only parts of the textures are displayed

**Comment by Mephiztopheles** [comment]
Figured out just now:  chests are affected too

**Comment by elfl0rd** [comment]
Workaround - restart your game. (have a problem while quitting?)
also please, attach client log from `%appdata%\..\LocalLow\Strange Loop Games\Eco` - output_log.txt

**Comment by Mephiztopheles** [comment]
I know the workaround :) 
I don't even have to quit, but disconnect
[output_log.txt](https://github.com/StrangeLoopGames/EcoIssues/files/2886272/output_log.txt)

**Comment by elfl0rd** [comment]
This is not related to storages, this related to 0 coordinate (are you passing trough it?)

**Comment by Mephiztopheles** [comment]
which coordinate? x/y/z? sometimes you're falling when you are stuck in walls or in mines when you jump and the roof hjas just one edge or something

**Comment by elfl0rd** [comment]
X or Z, Y vertical (you cannot reach 0 at)

**Comment by Mephiztopheles** [comment]
neither x or y... i just checked the coordinates and both are just on the other side of the world.
I'm around 600/300

**Comment by Mephiztopheles** [comment]
Maybe this is related to that?
![image](https://user-images.githubusercontent.com/6481850/53122041-ddbaa600-3555-11e9-8001-7856728aab26.png)

**Comment by Mephiztopheles** [comment]
[output_log.txt](https://github.com/StrangeLoopGames/EcoIssues/files/2886368/output_log.txt)

**Comment by elfl0rd** [comment]
Looks as broken chunk.

**Comment by Mephiztopheles** [comment]
looks like ntwork-issues. Now i got a picture of the stockpile:
![20190220212851_1](https://user-images.githubusercontent.com/6481850/53122435-e3fd5200-3556-11e9-9d8c-05cc5dc2bda1.jpg)

**Comment by Mephiztopheles** [comment]
wow... now when i pick up parts from that house, you see, it will not display the changes...

**Comment by marimbafarm** [comment]
I have seen similar behavior, where the contents appear first and then the stockpile containing them materializes later.

**Possible steps to reproduce**:
Given a 7 person town surrounded by river and farm fields, with houses and basements and stockpiles, I went on a long expedition to another continent and then came back to town to drop something off in a stockpile.

Other clues: it seems like it has a lot of trouble loading up the chunks along the way in a hurry (I have to wait for sections of bridge to appear so I can keep walking) and then the town feels especially slow to load.  So I can stand around and watch bits of terrain and other objects pop into view one by one.  I also haven't seen better than 10 fps while hanging around town for a few days.

**Expected behavior**:
Stockpile and its contents appear at the same time (preferably promptly when I'm standing next to it).

**Observed behavior**: 
Contents appear noticeably sooner than the stockpile, apparently while many nearby objects are also loading.

It seems like the stockpile contents can load before the stockpile object itself, so I can yank a few pieces of sandstone off the pile as though it was a freestanding rock pile, and then later (20 seconds?  5 minutes?  I wasn't paying attention) the stockpile boundary rope appears along the ground and the pile of stuff works like a regular stockpile again.  This has happened at least twice that I noticed.

**Mods**:
No mods.
